extensions/vscode-api-tests/src/singlefolder-tests/positron/runtime.test.ts

```diff
@@ -299,14 +299,11 @@ suite('positron API - runtime', () => {
 
 		managerDisposable.dispose();
 
-		// TODO: Unregistering a manager unregisters its runtimes, but doesn't remove them from
-		//       the list returned by positron.runtime.getRegisteredRuntimes. Is that a bug?
-		//       It also means that this test will currently fail if run out of order.
-		// await poll(
-		// 	getRegisteredRuntimes,
-		// 	(runtimes) => runtimes.length === 0,
-		// 	'test runtimes should be unregistered',
-		// );
+		await poll(
+			getRegisteredRuntimes,
+			(runtimes) => runtimes.length === 0,
+			'test runtimes should be unregistered',
+		);
```

Comment:
Okay @seeM I looked into this a bit and fixed the bug, below. I'm not sure it would have affected anything but it seems like a good change. Let me know what you think.

Reply:
Code looks good to me, thanks! Does the test pass locally? Open VSX is still down.

Reply:
Yep the whole test file passes locally! I won't merge until it's up again, though